reviews/layout/helpers.py

```diff
@@ -30,28 +30,36 @@ def render_pull_request_table(
     pull_requests = sorted(pull_requests, key=lambda x: x.updated_at, reverse=True)
 
     for pr in pull_requests:
-        approved = ""
-        if pr.approved == "APPROVED":
-            approved = "[green]Approved"
-        elif pr.approved == "CHANGES_REQUESTED":
-            approved = "[yellow]Changes Requested"
-
-        approved_by_others = "[green]Ready" if pr.approved_by_others else ""
+        # Formatting PR Title Column
+        title_colour = "[white]"
+        if "Security" in pr.title:
+            title_colour = "[yellow]"
```

Comment:
On second thought, `red` might be more appropriate here

```suggestion
            title_colour = "[red]"
```